src/Doctors/Domain/Models/Doctor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Doctors\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+// use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+/**
+ * @property int                          $id
+ * @property string                       $name
+ * @property \Carbon\CarbonImmutable|null $created_at
+ * @property \Carbon\CarbonImmutable|null $updated_at
+ *
+ * @method static \Database\Factories\DoctorFactory                    factory($count = null, $state = [])
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor whereName($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Doctor whereUpdatedAt($value)
+ *
+ * @mixin \Eloquent
+ */
+class Doctor extends Model
+{
+    protected $guarded = [];
```

Comment:
Id should be guarded since it's generated by the db

```suggestion
    protected $guarded = ['id'];
```